server/channels/app/email/email_batching.go

```diff
@@ -332,12 +334,27 @@ func (es *Service) sendBatchedEmailNotification(userID string, notifications []*
 				channelDisplayName = truncateUserNames(channel.DisplayName, 11)
 			}
 
+			postMessage := es.GetMessageForNotification(notification.post, translateFunc)
```

Comment:
I'm trying to think if there's similarly a good way to combine this with the code in `App.getNotificationEmailBody`. Could we perhaps move the rest of this (escaping the string, rendering the Markdown, and adding the hyperlinks) into `GetMessageForNotification`? That way, any code that needs to generate the HTML for a post can do it that with a single method call.

That would require passing the site URL, team ID, and team name into `GetMessageForNotification` so that we can generate the landing page URL and channel links, but we have all of those values both here and in `app/notification_email.go` where this is called